.github/workflows/sonar_analysis_kiegroup.yml

```diff
@@ -3,6 +3,14 @@ on:
   push:
     branches:
       - main
+    paths-ignore:
```

Comment:
I guess this could be added also to other workflows or ?

Reply:
@radtriste thank you, I tried to add the ignore-paths on places I think is applicable